tests/e2e/taqueria-plugin-taquito.spec.ts

```diff
@@ -0,0 +1,345 @@
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    // TODO: Currently there is no specific message for this type of error
+    // Issue associated https://github.com/ecadlabs/taqueria/issues/313
+    test.skip('Verify that taqueria taquito plugin will show proper error when configuration is wrong -> invalid pkh', async () => {
+        try {
+            // Environment test does not exist on default config.json
+            environment = "test"
+
+            // 1. Copy config.json and two michelson contracts from data folder to artifacts folder under taqueria project
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/hello-tacos.tz`);
+            execSync(`cp e2e/data/config-taquito-test-environment-invalid-faucet-pkh.json ${taqueriaProjectPath}/.taq/config.json`);
+
+            // 2. Run taq deploy on a network described in "test" environment
+            const stdoutDeploy = execSync(`taq deploy -e ${environment}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 3. Verify that proper error displays in the console
+            expect(stdoutDeploy).toContain("E_INVALID_PLUGIN_RESPONSE");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    // TODO: Currently there is no specific message for this type of error
+    // Issue associated https://github.com/ecadlabs/taqueria/issues/313
+    test.skip('Verify that taqueria taquito plugin will show proper error when configuration is wrong -> initial storage is not provided', async () => {
+        try {
+            // Environment test does not exist on default config.json
+            environment = "test"
+
+            // 1. Copy config.json and two michelson contracts from data folder to artifacts folder under taqueria project
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/hello-tacos.tz`);
+            execSync(`cp e2e/data/config-taquito-test-environment-invalid-initial-storage.json ${taqueriaProjectPath}/.taq/config.json`);
+
+            // 2. Run taq deploy on a network described in "test" environment
+            const stdoutDeploy = execSync(`taq deploy -e ${environment}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            // 3. Verify that proper error displays in the console
+            expect(stdoutDeploy).toContain("E_INVALID_PLUGIN_RESPONSE");
+            expect(stdoutDeploy).toContain('"No storage configured in your configuration file for hello-tacos.tz')
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    // Remove all files from artifacts folder without removing folder itself
+    afterEach(() => {
+        try {
+            const files = fs.readdirSync(`${taqueriaProjectPath}/artifacts/`);
+            for (const file of files) {
+                fs.unlinkSync(path.join(`${taqueriaProjectPath}/artifacts/`, file));
```

Comment:
Instead of unlink can you use fs.rmSync to remove the files?

Reply:
No, because rmSync also deletes the folder, that causes an issue for taqueria for the next test, because it cannot find the "artifacts" folder.